app/models/pxe_server.rb

```diff
@@ -213,4 +213,21 @@ def ensure_menu_list(menu_list)
   def self.display_name(number = 1)
     n_('PXE Server', 'PXE Servers', number)
   end
+
+  def self.verify_depot_settings_queue(userid, options)
+    task_opts = {
+      :action => "Verify PXE Server Credentials",
+      :userid => userid
+    }
+
+    queue_opts = {
+      :class_name  => name,
+      :method_name => "verify_depot_settings",
+      :queue_name  => "generic",
+      :role        => "ems_operations",
+      :args        => [options],
```

Comment:
is this a common pattern to put `options` into an Array before adding them to the queue?

Reply:
I think so. `args` is expecting an array.